cmd/livepeer/livepeer.go

```diff
@@ -390,7 +390,16 @@ func main() {
 			bigMaxGasPrice = big.NewInt(int64(*maxGasPrice))
 		}
 
-		client, err := eth.NewClient(ethcommon.HexToAddress(*ethAcctAddr), keystoreDir, *ethPassword, backend, ethcommon.HexToAddress(*ethController), EthTxTimeout, bigMaxGasPrice)
+		gpm := eth.NewGasPriceMonitor(backend, blockPollingTime, minGasPrice)
+		// Start gas price monitor
+		_, err = gpm.Start(ctx)
+		if err != nil {
+			glog.Errorf("error starting gas price monitor: %v", err)
```

Comment:
```suggestion
			glog.Errorf("Error starting gas price monitor: %v", err)
```

For consistency with other error messages.

Reply:
fixed in 0b64e17